File: Example ObjC/XYGamesDemo/ViewController.m
//
//  ViewController.m
//  XYGamesDemo
//
//  Created by Paul on 2023/11/22.
//

#import "ViewController.h"
#import "XYGamesKit-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    
    
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    
    UIViewController *vc = [XYManager createViewController];
    [self presentViewController:vc animated:true completion:^{
        
    }];
}


@end

File: Example/XYGamesKit/ViewController.swift
//
//  ViewController.swift
//  XYGamesKit
//
//  Created by 564057354@qq.com on 11/22/2023.
//  Copyright (c) 2023 564057354@qq.com. All rights reserved.
//

import UIKit
import XYGamesKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        super.touchesBegan(touches, with: event)
        
        let vc = YXManager.createViewController()
        self.navigationController?.pushViewController(vc, animated: true)
    }
    
}

